examples/active-active/README.md

```diff
@@ -0,0 +1,73 @@
+# EXAMPLE: Active-Active, External Services, Non-Airgap Installation of Terraform Enterprise
+
+## About This Example
+
+This example for Terraform Enterprise creates a TFE installation with the following traits.
+
+-  Active-Active
+-  External Services
+-  Non-Airgap
+-  m5.xlarge
+-  RHEL 7.9
+-  A privately accessible TCP load balancer with TLS pass-through
+-  An ubuntu based mitm proxy server with TLS termination 
+
```

Comment:
This list should still be readable, so maybe we can change the template to reflect such. Also, `Non-airgap` does not need to be called out, only if it _is_ airgapped.

```suggestion
-  Active/Active architecture
-  External Services production type
-  m5.xlarge virtual machine type
-  RHEL 7.9 
-  A privately accessible TCP load balancer with TLS pass-through
-  An ubuntu based mitm proxy server with TLS termination 

```

Reply:
creating PR